lib/react_client.dart

```diff
@@ -723,8 +771,9 @@ ReactDartComponentFactoryProxy2 _registerComponent2(ComponentFactory<Component2>
 
   /// Create the JS [`ReactClass` component class](https://facebook.github.io/react/docs/top-level-api.html#react.createclass)
   /// with custom JS lifecycle methods.
-  var reactComponentClass = createReactDartComponentClass2(_dartInteropStatics2, componentStatics, jsConfig2)
-    ..displayName = componentInstance.displayName;
+  var reactComponentClass =
+      createReactDartComponentClass2(_dartInteropStatics2, componentStatics, jsConfig2, filteredSkipMethods)
```

Comment:
#nit could we add `filteredSkipMethods` to the `JsComponentConfig2` object instead? It would be nice to minimize the number of positional arguments